src/laptop_battery_monitor/scripts/DockDriveActionClient.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python
+
+import roslib; roslib.load_manifest('kobuki_auto_docking')
+import rospy
+
+import actionlib
+from kobuki_msgs.msg import AutoDockingAction, AutoDockingGoal
+from actionlib_msgs.msg import GoalStatus
+from diagnostic_msgs.msg import DiagnosticArray
+
+
+class AutoDocking:
+    BATTERY_THRESHOLD = 4
```

Comment:
This is OK for now, but it would be good to make it a parameter configurable from the launch file.